modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -124,3 +191,39 @@ This command ensures that Runtime Fabric creates the necessary overrides for the
 The `rtfctl uninstall` command deletes any pull secret you specify during installation.
 
 When using `rtfctl` to perform a backup and restore procedure, the backups point to your local registry URL and pull secret. 
+--
+ 
+Helm::
++
+--
+. <<synchronize,Synchronize your container images>> to your local registry.
+. xref:install-helm.adoc#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager].
+. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
+. Obtain your private `docker-server`, `docker-username`, and `docker-password`.
+. Following the *Helm* path instructions in Runtime Manager, create a secret with your docker server name, username, and password by running the following command in your Unix shell:
++
+[source,copy]
+----
+kubectl create secret docker-registry rtf-pull-secret --namespace test-gke-rtf-qax-helm --docker-server=<docker-servername> --docker-username=<docker-username --docker-password=<docker-password> --docker-email=<docker-email>
```

Comment:
test-gke-rtf-qax-helm seems like a value we were using for testing

Reply:
@ndinu is it correct to change it  `--namespace <namespace-name>`?

Reply:
yes, please replace "test-gke-rtf-qax-helm" with "<rtf-namespace>"